source/perflib/strcasecmp.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef OSSL_PERFLIB_STRCASECMP_H
+# define OSSL_PERFLIB_STRCASECMP_H
+# pragma once
+
+extern const char *basename(const char *);
```

Comment:
You might want to edit the name 😉 

Also, FYI, Windows does have [`_stricmp`](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/stricmp-wcsicmp-mbsicmp-stricmp-l-wcsicmp-l-mbsicmp-l?view=msvc-170).

Reply:
Er, *this is strcasecmp.h*, changing the guard macro name was perhaps not the right move 😉